include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -308,4 +308,9 @@ static_assert(__cplusplus >= 201703L, "The Range support requires C++17 as minim
 #    define ONEDPL_ALLOW_DEFERRED_WAITING 0
 #endif
 
+//'present' macros
+// shift_left, shift_right; GCC 10; VS 2019 16.1
+#define _ONEDPL_CPP20_SHIFT_LEFT_RIGHT_PRESENT                                                                         \
+    (__cplusplus >= 202002L && ((_GLIBCXX_RELEASE && _GLIBCXX_RELEASE >= 10) || (_MSC_VER && _MSC_VER >= 1921)))
```

Comment:
```suggestion
    (__cplusplus >= 202002L && ((_GLIBCXX_RELEASE >= 10) || (_MSC_VER >= 1921)))
```

Everything else is redundant